server/config/migration/index.js

```diff
@@ -90,16 +85,16 @@ class Setup {
             console.log(error.message);
         });
 
-        const votes = `
+        const candidatesvotes = `
         CREATE TABLE IF NOT EXISTS votes (
             id SERIAL PRIMARY KEY,
             createdon date,
-            createdby integer,
-            office integer,
-            candidate integer
+            createdby INT,
+            office INT,
+            candidate INT
         );`;
 
-        this.pool.query(votes)
+        this.pool.query(candidatesvotes)
         .then((res) => {
             // console.log(res);
```

Comment:
if you are not using this console log, don't just comment it but remove it completely

Reply:
Yes I will do it definitely